src/GToolkit4Famix3-JavaScript/GtJSClass.class.st

```diff
@@ -1,5 +1,22 @@
 Class {
 	#name : #GtJSClass,
 	#superclass : #FAMIXClass,
+	#instVars : [
+		'accesses'
```

Comment:
This does not look appropriate here. In FAMIX, access denotes an association between a method and a variable. What specifically would you be interested in populating these accesses with?

Reply:
I'm not looking for extra functionality, just that as I was debugging issues with the downstream consumer (my GtTSImporter package), the dubugger rather consistently complained of GtJSClass not understanding `accesses` as the different nodes of the ast were traversed. Without this, a single file with a class would fail, after adding this it didn't. Let me create an example on my package that demonstrates the behavior and I'll reference it here.